### PR TITLE
mysqlbinlog: add page

### PR DESCRIPTION
**Pull request by einverne**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/mysqlbinlog.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Show the contents of the binary log file:
+
+`mysqlbinlog {{path/to/binlog}}`
+
+- Show binary log of a specific database:
+
+`mysqlbinlog --database {{database_name}} {{path/to/binlog}}`
+
+- Show binary log in a range:
```
Comment (kbdharun) [inline pages/common/mysqlbinlog.md:14]:
```suggestion
- Show binary log for a specific range:
```
Would this be better, your version is fine too.

File: pages/common/mysqlbinlog.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`mysqlbinlog --start-datetime='{{2022-01-01 01:00:00}}' --stop-datetime='{{2022-02-01 01:00:00}}' {{path/to/binlog}}`
+
+- Show binary log by using position:
```
Comment (navarroaxel) [inline pages/common/mysqlbinlog.md:18]:
```suggestion
- Show events from a binary log between specific positions:
```

File: pages/common/mysqlbinlog.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`mysqlbinlog --start-position={{100}} --stop-position={{200}} {{path/to/binlog}}`
+
+- Show binary log from the MySQL server on the given host:
```
Comment (navarroaxel) [inline pages/common/mysqlbinlog.md:22]:
```suggestion
- Show binary log from a MySQL server on the given host:
```

File: pages/common/mysqlbinlog.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# mysqlbinlog
+
+> Utility for processing MySQL binary log files.
+> More information: <https://dev.mysql.com/doc/refman/8.0/en/mysqlbinlog.html>.
+
+- Show the contents of the binary log file:
+
+`mysqlbinlog {{path/to/binlog}}`
+
+- Show binary log of a specific database:
```
Comment (navarroaxel) [inline pages/common/mysqlbinlog.md:10]:
```suggestion
- Show entries from a binary log for a specific database:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/mysqlbinlog.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Show the contents of the binary log file:
+
+`mysqlbinlog {{path/to/binlog}}`
+
+- Show entries from a binary log for a specific database:
+
+`mysqlbinlog --database {{database_name}} {{path/to/binlog}}`
+
+- Show binary log for a specific range:
```
Comment (navarroaxel) [inline pages/common/mysqlbinlog.md:14]:
```suggestion
- Show events from a binary log between specific dates:
```

File: pages/common/mysqlbinlog.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# mysqlbinlog
+
+> Utility for processing MySQL binary log files.
+> More information: <https://dev.mysql.com/doc/refman/8.0/en/mysqlbinlog.html>.
+
+- Show the contents of the binary log file:
```
Comment (navarroaxel) [inline pages/common/mysqlbinlog.md:6]:
```suggestion
- Show events from a specific binary log file:
```

File: pages/common/mysqlbinlog.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+`mysqlbinlog --start-datetime='{{2022-01-01 01:00:00}}' --stop-datetime='{{2022-02-01 01:00:00}}' {{path/to/binlog}}`
+
+- Show events from a binary log between specific positions:
+
+`mysqlbinlog --start-position={{100}} --stop-position={{200}} {{path/to/binlog}}`
+
+- Show binary log from a MySQL server on the given host:
+
+`mysqlbinlog --host={{host_name}} {{path/to/binlog}}`
```
Comment (navarroaxel) [inline pages/common/mysqlbinlog.md:24]:
```suggestion
`mysqlbinlog --host={{hostname}} {{path/to/binlog}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**